app/src/main/res/layout/group_list_item.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
```

Comment:
Only LinearLayouts support orientation, and I'm very sure CardView doesn't extend it.
```suggestion
```

Reply:
Yes, the parent was a LinearLayout and I forgot to remove it after changing it to CardView.